### DISCUSSION TITLE
Práctica 01: Abogabot en curso

### DISCUSSION THREAD

**Discussion by BarbaraMartel** [Launch-X-Latam/MisionFrontEnd#265] [Show and tell]
Buen día a todos. Hasta ahora, creo que la primera práctica es la que más me está costando, en referencia a los diseños del Wireframe UX y UI, ya que toman su tiempo. Cuando termine de desarrollarlos, lo actualizaré. Por lo pronto, me gustaría mostrar mi avance. Poniendo en práctica algunos de mis conocimientos previos en análisis y diseño de sistemas es que realicé la toma de los requerimientos y los diagramas presentados.

Aquí el enlace: [Práctica 01 - Abogabot](https://github.com/BarbaraMartel/PracticasMisionFrontEnd/tree/main/Practica-01)

**Comment by doguedogue** [comment]
Genial 🚀
